dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -42,6 +44,27 @@ public Response toolsOrgGet(
         return delegate.toolsOrgGet(organization, securityContext);
     }
 
+    @POST
+    @Path("/tools/entry/_search")
+    @UnitOfWork
+    @Produces({ "application/json" })
+    @ApiOperation(value = "Search the index of tools", notes = "This endpoint searches the index for all published tools and workflows. Used by utilities that expect to talk to an elastic search endpoint", tags = { "GA4GH", }, response = String.class)
+    @ApiResponses(value = { @ApiResponse(code = HttpStatus.SC_OK, message = "An elastic search result.", response = String.class) })
+    public Response toolsIndexSearch(@ApiParam(value = "elastic search query", required = true) String query, @Context UriInfo uriInfo, @Context SecurityContext securityContext) throws NotFoundException {
```

Comment:
It's a good point, I'm going to clean this up after the demo. 
The short answer is I expected it to be a GET but it turns out Elastic Search uses POST for this kind of operation
https://www.elastic.co/guide/en/elasticsearch/reference/2.4/_the_search_api.html